ptero_petri/api/v1/schemas/post_net.json

```diff
@@ -12,6 +12,7 @@
                         "oneOf": [
                             { "$ref": "#/definitions/actions/basic/convert_to_parent_color" },
                             { "$ref": "#/definitions/actions/basic/create_color_group" },
+                            { "$ref": "#/definitions/actions/basic/expire" },
```

Comment:
> Maybe pull this out into a transition definition that disallows outputs

I stopped short of fully defining in the schema a new transition type because there is no corresponding `expire` transition type in the python class hierarchy, which felt asymmetric.

I did create a limited definition of the basic transition (in the json schema) which I call `basic_expire`, that disallows outputs when the `expire` action is used on a basic transition.